<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRegisterTournament</class>
 <widget class="QDialog" name="DlgRegisterTournament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Please complete the following form for registering your tournament:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0">
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="leTnmtName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Organizing club:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCalendarWidget" name="cwFirst">
         <property name="gridVisible">
          <bool>false</bool>
         </property>
         <property name="dateEditEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Your name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="leMail"/>
       </item>
       <item row="2" column="2">
        <widget class="QCalendarWidget" name="cwLast"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Email address:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="leName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;First and last&lt;/p&gt;&lt;p&gt;tournament day:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="leClub"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tournament:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Note: we'll send a confirmation request to this email address.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOkay">
         <property name="text">
          <string>Register now...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leTnmtName</tabstop>
  <tabstop>leClub</tabstop>
  <tabstop>cwFirst</tabstop>
  <tabstop>cwLast</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>leMail</tabstop>
  <tabstop>btnOkay</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgRegisterTournament</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOkay</sender>
   <signal>clicked()</signal>
   <receiver>DlgRegisterTournament</receiver>
   <slot>onBtnOkayClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cwFirst</sender>
   <signal>selectionChanged()</signal>
   <receiver>DlgRegisterTournament</receiver>
   <slot>onFirstDayChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cwLast</sender>
   <signal>selectionChanged()</signal>
   <receiver>DlgRegisterTournament</receiver>
   <slot>onLastDayChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnOkayClicked()</slot>
  <slot>onFirstDayChanged()</slot>
  <slot>onLastDayChanged()</slot>
 </slots>
</ui>
